tests/plugins/test_flatpak_create_dockerfile.py

```diff
@@ -114,10 +118,18 @@ def test_flatpak_create_dockerfile(tmpdir, docker_tasker, config_name, breakage)
         assert breakage is None
         expected_exception = None
 
+    base_image = "registry.fedoraproject.org/fedora:latest"
+
     args = {
-        'base_image': "registry.fedoraproject.org/fedora:latest",
+        'base_image': base_image,
     }
 
+    if reactor_config_map:
+        workflow.plugin_workspace[ReactorConfigPlugin.key] = {}
+        workflow.plugin_workspace[ReactorConfigPlugin.key][WORKSPACE_CONF_KEY] =\
```

Comment:
Or could assign the dict, `= {`\n`WORKSPACE_CONF_KEY: ...` (also might make line wrapping easier)